blocks/card-carousel/card-carousel.css

```diff
@@ -0,0 +1,220 @@
+  padding: 0;
+}
+
+.career .career-slider{
+  margin-left: -1rem;
+}
+
+.career-slides {
+  display: flex;
+  overflow-x: scroll;
+  scroll-behavior: smooth;
+
+  /* We want to hide the scroll bar */
+  -ms-overflow-style: none; /* for Internet Explorer, Edge */
+  scrollbar-width: none; /* for Firefox */
+}
+
+/* Hide the scroll bar */
+.career-slides::-webkit-scrollbar {
+  /* for Chrome, Safari and Opera */
+  display: none;
+}
+
+.card-carousel .career-card {
+  flex-shrink: 0;
+  box-shadow: #3c465040 0 2px 12px 0;
+  margin: 1.5rem 3.5rem;
+  width: 20rem;
+  transition: all 0.15s ease-in-out;
+}
+
+.card-carousel.career .career-card {
+  margin: 1.5rem 0.5rem;
+  padding: 1rem;
+  width: var(--card-width);
+}
+
+.card-carousel .career-card:hover {
+  box-shadow: #3c465066 0 4px 16px 0;
+}
+
+.cards-card-image img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  object-position: 50% 0%;
+}
+
+.card-carousel .career-card a {
+  display: inline-flex;
+  flex-direction: column;
+  text-decoration: none;
+}
+
+.card-carousel .career-card a > div:nth-child(2) {
+  margin: 2rem 1rem;
+}
+
+.card-carousel.career .career-card a > div:nth-child(2) {
+  margin: 0;
+}
+
+card-carousel.career .career-card a > :not(picture) {
+  width: 100%;
+}
+
+.card-carousel .career-card picture {
+  align-self: center;
+  width: 14.375rem;
+  margin: 0 0 1.375rem;
+}
+
+.card-carousel .career-card picture img {
+  height: 9.5625rem;
+  width: 8.75rem;
+  padding-inline: 2.8rem;
+}
+
+.card-carousel .career-card blockquote {
+  color: var(--primary);
+  font-size: var(--body-font-size-m);
+  font-style: normal;
+  line-height: 24px;
+  margin: 0 0 19px;
+  max-height: 8rem;
+  overflow: hidden;
+  text-indent: 0;
+
+  /* Show ellipsis on multiline text */
+  display: -webkit-box;
+  -webkit-line-clamp: 5; /* number of lines to show */
+  -webkit-box-orient: vertical;
+}
+
+.card-carousel .career-card .career-card-bqc {
+  height: 8rem;
+  margin-bottom: 0.75rem;
+}
+
+.card-carousel .career-card h6 {
+  color: var(--light-black);
+  font-size: var( --heading-font-size-mxs);
+  text-transform: none;
+  margin-bottom: 6px;
+}
+
+.card-carousel .career-card p {
+  color: var(--light-black);
+  font-size: var(--body-font-size-s);
+  height: 2.5rem;
+  line-height: 21.6px;
+  margin-bottom: 0.75rem;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.card-carousel .career-card button {
+  background-color: unset;
+  color: var(--secondary);
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-m);
+  font-weight: var(--font-weight-medium);
+  justify-content: left;
+  height: auto;
+  padding: 0;
+  margin-block: 1rem;
+}
+
+.card-carousel .career-card button:hover {
+  text-decoration: underline;
+}
+
+.card-carousel .career-card button .icon-angle-right-blue {
+  height: 14px;
+  width: 10px;
+  margin-left: 6px;
+  margin-top: 2px;
+}
+
+.card-carousel .career-slides-navbar {
+  display: none;
+  margin-top: -0.5rem;
```

Comment:
Nit: I personally don't like negative values for margins and the sort, is there a way to maybe use `top` or `bottom` instead?